lib/matplotlib/backends/backend_wx.py

```diff
@@ -1474,25 +1474,6 @@ def updateButtonText(self, lst):
 }
 
 
-class SubplotToolWX(wx.Frame):
```

Comment:
API changes (even for "apparently unused" stuff) should nearly always go through a deprecation period unless it's really too hard to keep around.  Just decorate the class with `@deprecated("2.2")`.

Reply:
Done